core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -296,7 +280,7 @@ public Version minimumCompatibilityVersion() {
         final int bwcMinor;
         if (this.onOrAfter(Version.V_6_0_0_alpha1_UNRELEASED)) {
             bwcMajor = major - 1;
-            bwcMinor = 0; // TODO we have to move this to the latest released minor of the last major but for now we just keep
+            bwcMinor = 4;
```

Comment:
I am not sure we can / should do this. We guarantee that we are compatible with the latest minor of the previous major. That is `5.4` at this point. so I think this is correct? I think we also need to test against it though... it's a problem IMO but not a massive one.